account_payment_order/migrations/9.0.1.0.0/pre-migration.py

```diff
@@ -59,6 +59,25 @@
     ],
 }
 
+# They are being handled after module rename, so source module name is correct
```

Comment:
Shouldn't almost all the patch go into some `9.0.1.0.1/pre-migration.py` file so that people with `9.0.1.0.0` can upgrade and get the changes? 🤔

Reply:
No, this is a no-way. If you don't have before, now this is not going to work properly. So better to just leave it for new migrations. Also openupgradelib dependency for the rest of the people using this module without real migration prevents this.